modules/ytp-assets-common/src/less/ckan/upstream_ckan/module.less

```diff
@@ -151,8 +151,8 @@
     .actions {
         position: relative;
         float: right;
-        top: -10px;
-        right: -15px;
+        top: 15px;
+        right: 15px;
```

Comment:
Still different from the original, this file should not have any modifications.

Reply:
Sorry, I don't know what happened in my brain.